appcelerator/titanium_mobile
# [TIMOB-25703] Android: Added VideoPlayer "showsControls" property

**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25703

**Summary:**
- This boolean property shows/hides the overlaid video player controls.
- This is for parity with iOS, which added this property in Titanium 7.0.0.
- Did not remove "mediaControlStyle" property (for backward compatibility reasons) which can show/hide controls as well via different style constants. If the new "showsControls" property is false, then it will always hide the controls, regardless what the media style is set to (takes priority).

**Test:**
1. Build and run the below on an Android device.
2. Wait for the video view to start playing. (Requires an Internet connection to stream video.)
3. Tap on the video.
4. Verify that the video controls are **not** shown when tapped.

```javascript
var window = Ti.UI.createWindow({ fullscreen: true });
var videoView = Ti.Media.createVideoPlayer(
{
	url: "http://download.blender.org/peach/bigbuckbunny_movies/BigBuckBunny_640x360.m4v",
	autoplay: true,
	showsControls: false,
	scalingMode: Ti.Media.VIDEO_SCALING_ASPECT_FIT,
	width: Ti.UI.FILL,
	height: Ti.UI.FILL,
});
//videoView.showsControls = false;
window.add(videoView);
window.open();
```
